tests/ui/specs/SwitchItem/SwitchItem-specs.js

```diff
@@ -1,7 +1,7 @@
 const Page = require('./SwitchItemPage');
 
 // Skipping until SwitchItem is updated to handle the new Item structure
```

Comment:
I think we need to remove this comment because we activate it.

Reply:
You are correct. I removed this comment.